recipes/bgfx/all/conanfile.py

```diff
@@ -12,9 +12,6 @@
 from pathlib import Path
 import os
 
-required_conan_version = ">=1.50.0"
```

Comment:
I was suggesting to remove this in test package, to main conanfile. You know you can accept suggestions as commits?

Reply:
Oops, I'm blind sometimes. And I actually didn't know that, found it now!